install/operator/pkg/cmd/internal/olm/olm.go

```diff
@@ -50,7 +50,7 @@ func New(parent *internal.Options) *cobra.Command {
 }
 
 func (o *Olm) generate() (err error) {
-	conf, err := configuration.GetProperties(context.TODO(), configuration.TemplateConfig, o.ClientTools(), &v1beta1.Syndesis{})
+	conf, err := configuration.GetProperties(context.TODO(), configuration.TemplateConfig, nil, &v1beta1.Syndesis{})
```

Comment:
For my understanding (sure it makes sense), can you explain why this change?

Reply:
Because when I run command I dont need to connect to any cluster. This command is run offline.